.vib/jaeger/vib-publish.json

```diff
@@ -0,0 +1,54 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/jaeger"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
```

Comment:
You also need to edit the .github/workflows/cd-pipeline.yaml and add a line for jaeger

Reply:
Done [78740e9](https://github.com/bitnami/charts/pull/13480/commits/78740e982d8d52afa6159d44b51eb471d48c889e)